src/wp-includes/class-wp-scripts.php

```diff
@@ -797,6 +804,20 @@ private function get_intended_strategy( $handle ) {
 		return isset( $script_args['strategy'] ) ? $script_args['strategy'] : false;
 	}
 
+	/**
+	 * Get the strategy assigned during script registration.
+	 *
+	 * @param string $handle   The script handle.
+	 * @param string $position Position of the inline script.
+	 *
+	 * @return bool True if script present. False if empty.
+	 */
```

Comment:
```suggestion
	/**
	 * Check if a script has a non standalone inline script associated with it.
	 *
	 * @param string $handle   The script handle.
	 * @param string $position Position of the inline script.
	 *
	 * @return bool True if script present. False if empty.
	 */
```

@kt-12 I think this docblock was incorrectly described? It does not obtain a strategy.